### ISSUE TITLE
Bulk Edits: OSG & OE Cleanup

### ISSUE THREAD

**Issue by simholt** [osulp/Scholars-Archive#1832]
### Descriptive summary

Cleanup in multiple fields for Oregon Sea Grant and Oregon Explorer items. I collected all the changes in one tab:

- [x] [Bulk Edits OSG & OE](https://docs.google.com/spreadsheets/d/1Kwqfsr53qlcp20Hol7VX-NPme6ToKmzsLZpFEaGjT-k/edit#gid=309272308&range=A2461)

This includes removing specific entries from the Additional Information field with "-". Greg did this last time & is going to update the wiki. (I hope I listed them correctly.)

**Comment by luisgreg99** [comment]
@simholt I've updated the wiki [Bulk Metadata Modifications
](https://github.com/osulp/Scholars-Archive/wiki/Bulk-Metadata-Modifications) with notes and examples for the `-` operator. 

I checked your list, and I don't think it would work for ordered fields like `additional_information`. So far it only works for multi-value unordered fields where the expected type is `text`, but I think we can extend it in the script to support ordered fields.
